featurebyte/models/tile_registry.py

```diff
@@ -0,0 +1,83 @@
+"""
+TileModel document model
+"""
+from __future__ import annotations
+
+from typing import List, Optional
+
+from datetime import datetime
+
+import pymongo
+from pydantic import StrictStr
+
+from featurebyte.models.base import (
+    FeatureByteCatalogBaseDocumentModel,
+    PydanticObjectId,
+    UniqueValuesConstraint,
+)
+from featurebyte.schema.common.base import BaseDocumentServiceUpdateSchema
+
+
+class TileModel(FeatureByteCatalogBaseDocumentModel):
+    """
+    TileModel document
+    """
+
+    feature_store_id: PydanticObjectId
+    tile_id: StrictStr
+    aggregation_id: StrictStr
+
+    tile_sql: StrictStr
+    entity_column_names: List[StrictStr]
+    value_column_names: List[StrictStr]
+    value_column_types: List[StrictStr]
+
+    frequency_minute: int
+    time_modulo_frequency_second: int
+    blind_spot_second: int
```

Comment:
Should we impose some constraint here? Like the value must be greater than 0 and less than certain value?

Reply:
These fields should have been validated upstream, but I think it makes sense to impose the same constraints here.